src/main/java/racingcar/domain/CarPosition.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.domain;
+
+import java.util.Objects;
+
+public class CarPosition {
+    public static final int INIT_POSITION = 0;
+    public static final String NEGATIVE_VALUE_MESSAGE = "position은 음수 값을 가질 수 없습니다.";
+
+    private int position;
+
+    public CarPosition() {
+        this(INIT_POSITION);
+    }
+
+    public CarPosition(final int position) {
+        if (position < INIT_POSITION) {
+            throw new IllegalArgumentException(NEGATIVE_VALUE_MESSAGE);
+        }
+        this.position = position;
+    }
+
+    public CarPosition plus() {
+        position++;
+        return this;
+    }
```

Comment:
```suggestion
    public CarPosition plus() {
        return new CarPosition(position + 1);
    }
```

수정된 코드는 기존 코드와 어떤 차이가 있을까요?